pkg/sentry/kernel/thread_group.go

```diff
@@ -286,6 +286,16 @@ func (tg *ThreadGroup) SignalHandlers() *SignalHandlers {
 	return tg.signalHandlers
 }
 
+// IsReaperLocked check tg is reaper of the namespace.
+//
+// Preconditions: ts.mu must be locked for writing.
```

Comment:
Why do we need to hold the lock for *writing*?  It seems like reading should be enough.

Reply:
Thanks, fix code changed in new patch.